supabase/migrations/20240415132809_add_creation_timestamp_to_parcels.sql

```diff
@@ -0,0 +1,33 @@
+alter table "public"."parcels" add column "created_at" timestamp with time zone not null default (now() AT TIME ZONE 'utc'::text);
+
+update "public"."parcels" set "created_at"='2023-12-31 12:00:00+00';
```

Comment:
Maybe I'm confused as to the point in which this file runs, but should this be  `....WHERE "created_at" IS NULL;`?

Reply:
This runs when the migration happens. When this is merged to dev for example, the outstanding migrations (just this file) runs against the dev db, which would
- create the created_at column
- set created_at to a certain time.

Since we're creating the column just before, we don't need to worry about `....WHERE "created_at" IS NULL;`

Reply:
makes sense, thanks